src/WebformBuilder.js

```diff
@@ -441,6 +441,7 @@ export default class WebformBuilder extends Webform {
     }
     this.addClass(this.element, 'builder-paste-mode');
     const copy = _.cloneDeep(component.schema);
+    BuilderUtils.uniquify(this._form, copy);
```

Comment:
This should probably occur on the ```pasteComponent``` method since I can see a problem where you could add another component between the point where you copied it and pasted it and then you would have another collision.

Reply:
Also, you can paste multiple times which would cause a collision.